core/src/main/java/org/elasticsearch/index/query/GeoShapeQueryBuilder.java

```diff
@@ -32,22 +53,27 @@
 public class GeoShapeQueryBuilder extends AbstractQueryBuilder<GeoShapeQueryBuilder> {
 
     public static final String NAME = "geo_shape";
+    public static final String DEFAULT_SHAPE_INDEX_NAME = "shapes";
+    public static final String DEFAULT_SHAPE_FIELD_NAME = "shape";
+    public static final ShapeRelation DEFAULT_SHAPE_RELATION = ShapeRelation.INTERSECTS;
 
-    static final GeoShapeQueryBuilder PROTOTYPE = new GeoShapeQueryBuilder(null, null);
+    static final GeoShapeQueryBuilder PROTOTYPE = new GeoShapeQueryBuilder(null, (BytesReference) null);
```

Comment:
can we pass empty string and empty bytes ref to this so it's a valid ctor? we should fail if these args are null